src/components/MarkdownProvider/components/PackageDescription.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+
+export interface IPackage {
+  version: string;
+  name: string;
+  description: string;
+}
+
+export const PackageDescription: React.FC<{ data: IPackage }> = ({ data }) => {
+  return (
+    <ul>
+      <li>Name: {data.name}</li>
+      <li>Version: {data.version}</li>
+      <li>Description: {data.description}</li>
+    </ul>
```

Comment:
Can this be a Garden [`UnorderedList`](https://garden.zendesk.com/react-components/typography/#unorderedlist) with `style={{margin: 0, listStyleType: 'none'}}`?

Reply:
This area isn't designed in any way. It's just a placeholder. I've updated to use our lists though.